src/projects/TodoList/index.js

```diff
@@ -13,23 +15,33 @@ class TodoList extends React.Component {
   constructor() {
     super();
     this.addItem = this.addItem.bind(this)
+    this.removeItem = this.removeItem.bind(this)
     this.state = {
       todos: ['Take out the bins', 'Put a wash load on', 'Learn React'],
-      newItem: ''
+      newItem: '',
+      removedIndex: ''
     }
   }
   addItem() {
     const newTodos = this.state.todos
     newTodos.push(this.state.newItem)
     this.setState({ todos: newTodos, newItem: '' })
   }
+  removeItem() {
+    let removedItem = this.props.key
+    this.setState({ removedIndex: removedItem })
+    const removedTodos = this.state.todos
+    removedTodos.splice(this.state.todos[this.state.removedIndex], 1)
```

Comment:
```js
array.splice(i, numberOfItemsToRemove)
```

`this.state.todos[this.state.removedIndex]` will equal a string? This should just be the index